packages/init/src/utils/validateRepositoryName.ts

```diff
@@ -33,5 +33,12 @@ export async function validateRepositoryName(
     return Promise.reject(new Error(msg));
   }
 
-  return client.domainExist(domain).catch(() => false);
+  return client
+    .domainExist(domain)
+    .then((isAvailable) =>
+      isAvailable
+        ? true
+        : Promise.reject(new Error(`${name} is already in use`))
```

Comment:
should throw no?
Promise.resolve(Prosmise.reject(...))

Reply:
Promise.reject will act as a throw, we'll be able to catch it later.
I kept the same behavior as there was before.

Reply:
I mean in that case, we don't catch it so inquirer knows it's an error (tested it myself 😉 )

Reply:
hey @MarcMcIntosh , made a small test 
<img width="561" alt="image" src="https://user-images.githubusercontent.com/41187899/174751225-ce6c6e91-7891-4f41-be9e-369641cb62e7.png">

<img width="499" alt="image" src="https://user-images.githubusercontent.com/41187899/174751504-eb479791-8249-4750-92c7-51b8c464b819.png">

I put back the error message just in case but I believe that settle this 😄